test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -695,7 +742,7 @@ public void testLogEntrySyncValidCrossTablesWithTriggerTimeout() throws Exceptio
         expectedAckMessages =  NUM_KEYS*WRITE_CYCLES;
 
         testConfig.clear().setDropMessageLevel(2);
-        startLogEntrySync(crossTables, WAIT.ON_ERROR);
+        startLogEntrySync(crossTables, WAIT.ON_TIMEOUT_ERROR);
```

Comment:
Why did this change?

Reply:
It is much clear about what each test is wait on.